test/mailers/previews/webhooks/outgoing/endpoint_mailer_preview.rb

```diff
@@ -0,0 +1,23 @@
+class Webhooks::Outgoing::EndpointMailerPreview < ActionMailer::Preview
+  def deactivation_limit_reached
+    endpoint = Webhooks::Outgoing::Endpoint.last ||
+      Webhooks::Outgoing::Endpoint.new(
```

Comment:
We'll notify about the last or new endpoint? 🤔 

How is it guaranteed that the last endpoint is the one failing?

`new` doesn't make sense to me at all, not sure why it's there. 

Imo, it should have the endpoint passed to it in the params?

Same for the `deactivated` method below.